roles/django/tasks/install_local_settings.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Creates a new `local_settings.py` file based on a template.
+# https://docs.djangoproject.com/en/dev/ref/settings/
+
+- name: install django local settings
+  template:
+    src: "{{ django_local_settings_template }}"
+    dest: "{{ django_app_path }}/{{ django_app }}/local_settings.py"
```

Comment:
This path needs to be configurable for apps that put them in a non-standard location (flask prototypes, projects with split-settings)